packages/playwright-core/src/utils/registry.ts

```diff
@@ -491,7 +491,25 @@ export class Registry {
       executablePathOrDie: (sdkLanguage: string) => executablePath(sdkLanguage, true)!,
       installType: install ? 'install-script' : 'none',
       validateHostRequirements: () => Promise.resolve(),
-      _install: install,
+      _install: install ? async (forceReinstall: boolean) => {
+        const execPath = executablePath('', false);
+        if (!forceReinstall && execPath) {
+          const { langName } = getClientLanguage();
+          const command = buildPlaywrightCLICommand(langName, 'install --force ' + name);
+          throw new Error('\n' + wrapInASCIIBox([
+            `${name} is already installed on the system.`,
+            `To re-install ${name}:`,
```

Comment:
maybe good to outline there that a recent version gets installed and its not just a re-install.